arkouda/util.py

```diff
@@ -248,7 +250,17 @@ def attach(name: str, dtype: str = "infer"):
         dtype = repMsg.split()[2]
         repMsg = repMsg[len(repType) + 1 :]
         if dtype == "str":
-            return Strings.from_return_msg(repMsg)
+            s = Strings.from_return_msg(repMsg)
+            registry = list_registry()
+            if f"{name}_bytes" in registry:
+                s._bytes = create_pdarray(
+                    cast(str, generic_msg(cmd="attach", args={"name": name + "_bytes"}))
+                )
+            if f"{name}_offsets" in registry:
+                s._offsets = create_pdarray(
+                    cast(str, generic_msg(cmd="attach", args={"name": name + "_offsets"}))
+                )
```

Comment:
I don't love us double defining the name for bytes/offsets in 2 places and 2 different ways (one with f strings and one with adding strings. I don't think see any issues but it seems error prone (one gets updated but not the other). Maybe do something like
```python
registry = list_registry()
bytes_name, offsets_name = f"{name}_bytes", f"{name}_offsets"
if bytes_name in registry:
    s._bytes = create_pdarray(
        cast(str, generic_msg(cmd="attach", args={"name": bytes_name}))
    )
if offsets_name in registry:
    s._offsets = create_pdarray(
        cast(str, generic_msg(cmd="attach", args={"name": offsets_name}))
    )
```